pkgs/development/compilers/gcc/6/default.nix

```diff
@@ -120,6 +120,11 @@ stdenv.mkDerivation ({
     repo = "gcc-vc4";
     rev = "e90ff43f9671c760cf0d1dd62f569a0fb9bf8918";
     sha256 = "0gxf66hwqk26h8f853sybphqa5ca0cva2kmrw5jsiv6139g0qnp8";
+  } else if stdenv.targetPlatform.isRedox then fetchFromGitHub {
+    owner = "redox-os";
+    repo = "gcc";
+    rev = "f360ac095028d286fc6dde4d02daed48f59813fa";
```

Comment:
This `rev` at least isn't the master branch. Where is this coming from? Maybe add a comment.

Reply:
Fixed. I added a comment stating that the branch is `redox`